newrelic/exporter_test.go

```diff
@@ -165,7 +166,10 @@ func TestEndToEndTracer(t *testing.T) {
 		}
 		depth := numSpans - n
 		ctx, span := tracer.Start(ctx, fmt.Sprintf("Span %d", depth))
-		span.SetAttributes(label.Int("depth", depth))
+		span.SetAttributes(
+			attribute.Int("depth", depth),
+			semconv.ServiceNameKey.String(serviceName),
```

Comment:
Hi @akulnurislam ! This one has us scratching our heads a bit. What has changed to require us to add this serviceName in order for the test to pass?

Reply:
@RichVanderwal ya hahaha, I saw `NewExporter` doesn't have any setAttributes.

If I look `NewExportPipeline` there is set attribute serviceName for otel controller.

https://github.com/newrelic/opentelemetry-exporter-go/blob/e563425c51c9e256b329c37e72192baac0396aca/newrelic/exporter.go#L111-L113

Reply:
@RichVanderwal I've changed set attribute into `NewTracerProvider`